lib/rubocop/cop/ezcater/rspec_require_feature_flag_mock.rb

```diff
@@ -0,0 +1,53 @@
+module RuboCop
+  module Cop
+    module Ezcater
+      # Enforce usage of our `mock_feature_flag` helpers over explicit usage
+      # of allow(FeatureFlag) mocking.
+      #
+      # @example
+      #
+      #   # good
+      #   mock_feature_flag("MyFeatureFlag", true)
+      #   mock_feature_flag("MyFeatureFlag", { user: current_user }, true)
+      #
+      #   # bad
+      #   allow(FeatureFlag).to receive(:is_active?).and_return(true)
+      #   allow(FeatureFlag).to receive(:is_active?).with("MyFeatureFlag").and_return(true)
+      #   allow(FeatureFlag).to receive(:is_active?).with("MyFeatureFlag", user: current_user).and_return(true)
+      class RspecRequireFeatureFlagMock < Cop
+        MSG = "Please use the `mock_feature_flag` helper over " \
+          "`allow(FeatureFlag)` directly.".freeze
```

Comment:
Re-word suggestion "Use the `mock_feature_flag` helper instead of mocking `FeatureFlag.is_active?`"?